### DISCUSSION TITLE
ComparisonError

### DISCUSSION THREAD

**Discussion by adbo** [argoproj/argo-cd#6496] [Q&A] [answered]
Hi,
After creating an app I received
`rpc error: code = Unknown desc = Get "https://github.com/..... context deadline exceeded (Client.Timeout exceeded while awaiting headers)`
Creating replicas of argocd-repo-server  didn't help. When I try to exec git clone on argocd-repo-server I received "Could not resolve host: github.com".

Im using argocd with k3s. K3s is installed on VM with Suse. I'm using virtualbox and vagrant to run VM.

How to handle this issue?

**Comment by jannfis** [comment]
It seems that the pods in your cluster cannot resolve DNS, or access resources on the Internet at all.

Please check your cluster configuration.

**Reply by adbo** [reply]
I need to add that this issue occur randomly. Today morning everything was working, now I have again this timeout error. Do you have any idea what is going on and how I can debug it?

**Comment by adbo** [comment, marked as answer]
OK, I found solution:
`kubectl -n kube-system rollout restart deployment coredns`
